packages/@aws-cdk-containers/ecs-service-extensions/test/appmesh.test.ts

```diff
@@ -516,9 +504,7 @@ describe('appmesh', () => {
     greeterService.connectTo(greetingService);
 
     // THEN
-    expect(stack).toHaveResource('AWS::ECS::TaskDefinition');
-
-
+    Template.fromStack(stack).hasResource('AWS::ECS::TaskDefinition', Match.anyValue());
```

Comment:
This could also be `resourceCountIs( , 1)`. Which one is better?

Reply:
So the slight difference here is that `resourceCountIs(,1)` ensures that there is only 1 resource `AWS::ECS::TaskDefinition`. I'm 99% sure that when I was migrating, this test had multiple task definitions, which is why I migrated it this way.

Maybe its worth thinking about adding comparisons to `resourceCountIs`? Something like `resourecCountIs(, Match.greaterThan(1))`.

Reply:
Ah okay, thanks for the explanation. Make sense. I think it's worth opening a feature request to Assertions for this one.